third_party/nvfuser/CMakeLists.txt

```diff
@@ -132,7 +132,8 @@ list(APPEND NVFUSER_SRCS
 )
 
 add_library(${NVFUSER_CODEGEN} SHARED ${NVFUSER_SRCS})
-torch_compile_options(${NVFUSER_CODEGEN})
+# Note: for some reason, torch_compile_options gives us segfaults when we run test_jit
+#torch_compile_options(${NVFUSER_CODEGEN})
```

Comment:
I wonder what the impact of this change would be. Was this the only place compile options were configured? Does removing it mean options like `-Wall` is not used?